### PR TITLE
[Azure Stack] Updates to User-Subscriptions Spec

### PR DESCRIPTION
**Pull request by viananth**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by njuCZ** [comment]
@viananth please fix the CI error

File: specification/azsadmin/resource-manager/user-subscriptions/readme.md [hunk 1]
```diff
@@ -101,29 +101,3 @@ These settings apply only when `--tag=package-2015-11-01 --python` is specified
 ``` yaml $(tag) == 'package-2015-11-01' && $(python)
 namespace: azure.mgmt.subscriptions.v2015_06_01_preview
 ```
-
-## Multi-API/Profile support for AutoRest v3 generators
```
Comment (bganapa) [inline specification/azsadmin/resource-manager/user-subscriptions/readme.md:105]: this should be in the readme.md as well

File: specification/azsadmin/resource-manager/user-subscriptions/readme.azsautogen.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# Subscriptions Admin
```
Comment (bganapa) [inline specification/azsadmin/resource-manager/user-subscriptions/readme.azsautogen.md:1]: this file is not needed, pls remove. The private repo does nto refer to this file

**Comment by viananth** [comment]
> @viananth please fix the CI error

@njuCZ CI is reporting this as an error, but there are no errors in example validation:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=340779&view=logs&j=8d2f9c49-cd83-5a9d-f3fe-2fd651afa742&t=43324f1f-6724-5d61-785f-f0ac9fd256a1&l=28

**Review by bganapa — APPROVED** [review]

**Comment by viananth** [comment]
@njuCZ I reviewed the breaking changes and they are expected. Could you please take a look and let me know if you have any feedback?

**Comment by viananth** [comment]
@njuCZ could you pls approve this PR if it looks good to you? @bganapa as FYI

**Comment by njuCZ** [comment]
@viananth Sorry for missing this thread.  According to our latest policy:  Per [Retirement of Preview](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews) guidance, products in Preview for over one year need to move to GA or retire; require CVP exemption after 1 year in preview

**Comment by viananth** [comment]
@bganapa FYI

**Comment by bganapa** [comment]
@njuCZ This is another Azure Stack Admin API swagger spec where we are planning to move all our specs to stable in Fe timeframe. Could you please approve this PR and proceed to merge. Thank you!

**Review by njuCZ — APPROVED** [review]

**Pull request merged**